Plugins/GRPCProtobufGeneratorCommand/Plugin.swift

```diff
@@ -70,21 +122,15 @@ struct GRPCProtobufGeneratorCommandPlugin: CommandPlugin {
         outputDirectory: outputDirectory
       )
 
-      if commandConfig.verbose || commandConfig.dryRun {
-        printProtocInvocation(protocPath, arguments)
-      }
+      try executeProtocInvocation(
+        executableURL: protocPath,
+        arguments: arguments,
+        verbose: commandConfig.verbose,
+        dryRun: commandConfig.dryRun
+      )
+
       if !commandConfig.dryRun {
-        let process = try Process.run(protocPath, arguments: arguments)
-        process.waitUntilExit()
-
-        if process.terminationReason == .exit, process.terminationStatus == 0 {
-          if commandConfig.verbose {
-            Stderr.print("Generated gRPC Swift files for \(inputFiles.joined(separator: ", ")).")
-          }
-        } else {
-          let problem = "\(process.terminationReason):\(process.terminationStatus)"
-          Diagnostics.error("Generating gRPC Swift files failed: \(problem)")
-        }
+        Stderr.print("Generated gRPC Swift files for \(inputFiles.joined(separator: ", ")).")
```

Comment:
I don't think we should print this unless `verbose` (rationale: I called the command with the input files, I know what they are.)